README.md

```diff
@@ -31,7 +31,16 @@ It's symfony website-skeleton project with some additional tools to validate cod
 1. Clone this project
 2. Run `composer install`
 3. Run `yarn install`
-4. Run `yarn encore dev` to build assets
+4. Run `yarn encore dev` to build assets  
+*Option :*
+5. If you encounter this error message : <u>"Only the ‘en’ locale is supported. Please install the ‘intl’ extension for full localization capabilities.</u>"  
+Please read [Twig > Docs > Filters > format_datetime](https://twig.symfony.com/doc/3.x/filters/format_datetime.html)  
+5.0. Run `composer recipes` to check if *twig/intl-extra, twig/extra-bundle* are available, if not :  
+5.1. Run `composer require [twig/intl-extra] [twig/extra-bundle]`  
+5.2. Install php7.4-intl module : `sudo apt install php7.4-intl` (If you run php7.4...)  
+5.3. Restart apache : `service apache2 restart` (Is it really necessary?)
```

Comment:
Depends if you run apache or not...
Do you really need the intl component as you don't have i18n in your project for now ?

Reply:
The `intl` component is needed to give a local format to the dates that will be displayed in the views.
Please read :
https://twig.symfony.com/doc/3.x/filters/format_datetime.html

Reply:
But now, it's in the composer.json/lock so the composer install command will install the intl-extra